src/components/ProfileSkeleton.tsx

```
@@ -0,0 +1,38 @@
+      </div>
+
+      {/* Profile details */}
+      <div className="grid gap-4 sm:grid-cols-2">
+        <Skeleton className="h-12 w-full rounded-md" />
+        <Skeleton className="h-12 w-full rounded-md" />
+      </div>
+
+      <span className="sr-only">Loading profile...</span>
+    </div>
+  );
+}
```

ESLint

[error] Insert `⏎`

(prettier/prettier)